### ISSUE TITLE
browser.driver vs browser. Which one to use?

### ISSUE THREAD

**Issue by demisx** [angular/protractor#634]
Can someone please explain in plain English (or point to docs) on when I should use `browser.driver` and when `browser` when calling methods? For example:

```
browser.get() or browser.driver.get()?
browser.getAllWindowHandles() or browser.driver.getAllWindowHandles()?
browser.getCurrentUrl() or browser.driver.getCurrentUrl()?
```

Thanks.

**Comment by ryancat** [comment]
Hi @demisx do you have the answer to your question on April 1, 2014? I am confused on this as well.

**Comment by demisx** [comment]
@ryancat Hi there. As far as I understood, `browser.driver.x` is for interaction with non-Angular pages and `browser.x` methods are wrappers for interacting with Angular pages.

**Comment by MarkAPhillips** [comment]
Just to add to this - I find that ```browser.driver``` works better on AngularJS apps that take time to sync. I have tried both and for some reason ```browser.driver``` is more reliable.

I think more clarity needs to added in relation to this as why not just use ```browser.driver``` as it works on both Angular and non-Angular apps ???? and therefore eliminates this confusion.
